### ISSUE TITLE
bug: FirebaseAuthentication.signInWithFacebook() does not resolve

### ISSUE THREAD

**Issue by morethal** [capawesome-team/capacitor-firebase#243]
**Plugin(s):**

```
"@capacitor-firebase/authentication": "^1.2.0",
"@capacitor/android": "^4.4.0",
"@capacitor/core": "^4.4.0",
"firebase": "9.12.1",
```

**Platform(s):**
Android

**Current behavior:**
The promise `await FirebaseAuthentication.signInWithFacebook();` does not resolve.

**Expected behavior:**
It resolves and the user is logged in, or if there is an error, it should be thrown.

**Steps to reproduce/Related code:**
Essentially I followed the sample code:

```
export function whichAuth() {
    if (!firebaseAuth) {
            // https://github.com/capawesome-team/capacitor-firebase/issues/230#issuecomment-1307494032
            firebaseAuth = initializeAuth(firebaseApp, {
                persistence: indexedDBLocalPersistence,
            });
    }
    return firebaseAuth;
}

async function facebookLogin(){
  const auth = whichAuth();
  const result = await FirebaseAuthentication.signInWithFacebook();
  // This is never exectued... 
  console.log(result);
}
```

Note: While not necessary, I removed all code occurrences of `getAuth()` to avoid any side effects.

**Capacitor doctor:**

```
Latest Dependencies:

  @capacitor/cli: 4.4.0
  @capacitor/core: 4.4.0
  @capacitor/android: 4.4.0
  @capacitor/ios: 4.4.0

Installed Dependencies:

  @capacitor/ios: not installed
  @capacitor/core: 4.4.0
  @capacitor/android: 4.4.0
  @capacitor/cli: 4.4.0
```

Also see: https://github.com/capawesome-team/capacitor-firebase/issues/230#issuecomment-1307436589

**Comment by robingenz** [comment]
Hi @morethal,
please provide a Minimal, Reproducible Example (see [How to create a Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example)) so I can debug the issue.

**Comment by morethal** [comment]
Thanks for the swift reply. Is there anything you are missing for a repro or is there anything I can check on my side? 

I can check if I can create a fresh project and just add the sample code so this can be shared easily.

**Comment by robingenz** [comment]
> I can check if I can create a fresh project and just add the sample code so this can be shared easily.

Yes, that's what I need.

**Comment by morethal** [comment]
Thanks, when I created a repro I could see that the repro was actually working as expected. When comparing the differences I could identify that the error seems to be related to:

`implementation "androidx.appcompat:appcompat:1.5.1"`

It works fine with:

`implementation "androidx.appcompat:appcompat:1.4.2"`

When I change the version, it break again.

This seems to be a known bug here:
https://github.com/facebook/facebook-android-sdk/issues/1100
https://github.com/facebook/facebook-android-sdk/issues/1095

This seems to be fixed upstream with `Facebook SDK sdk-version-14.1.0`

I was able to locally fix this by changing the Facebook Login version via setting this in the variables.gradle:
```
facebookLoginVersion = '15.1.0'
```

Any chance we might be able to up the version here as well?
